packages/account/src/providers/provider.ts

```diff
@@ -739,12 +739,12 @@ Supported fuel-core version: ${supportedVersion}.`
    * @param sendTransactionParams - The provider send transaction parameters (optional).
    * @returns A promise that resolves to the transaction response object.
    */
-  // #region Provider-sendTransaction
   async sendTransaction(
     transactionRequestLike: TransactionRequestLike,
     { estimateTxDependencies = true }: ProviderSendTxParams = {}
   ): Promise<TransactionResponse> {
     const transactionRequest = transactionRequestify(transactionRequestLike);
+    // #region Provider-sendTransaction
```

Comment:
@maschad Why this code snippet region was placed in a different line?

Reply:
In my opinion it better articulates the process of the dependency estimation,  for example the lines related to `transactionRequest` are irrelevant.

# Before

<img width="739" alt="Screenshot 2024-08-29 at 1 35 30 PM" src="https://github.com/user-attachments/assets/40233b17-6c26-4528-8ebd-484a5b560a10">


# After

<img width="808" alt="Screenshot 2024-08-29 at 1 37 56 PM" src="https://github.com/user-attachments/assets/dbfda416-55f4-4d77-a5c9-62244109b471">